packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "courage-for-signin-widget",
+  "version": "1.0.0",
+  "description": "generates a Courage version for signin-widget",
+  "main": "src/OktaForSigninWidget.js",
```

Comment:
Okta for Courage for the widget?

`CourageForSigninWidget` might be more clear.